### PR TITLE
GPP passthrough for cookiesync

### PR DESCRIPTION
**Pull request by hhhjort**
Allows GPP strings to pass through cookie sync endpoints.

### CODE REVIEW

**Review by guscarreon — COMMENTED** [review]
~Do we need to pass GPP info into `rx.Privacy`?~
```
188     rx := usersync.Request{
189         Bidders: request.Bidders,
190         Cooperative: usersync.Cooperative{
191             Enabled:        (request.CooperativeSync != nil && *request.CooperativeSync) || (request.CooperativeSync == nil && c.config.UserSync.Cooperative.EnabledB
192             PriorityGroups: c.config.UserSync.Cooperative.PriorityGroups,
193         },
194         Limit: request.Limit,
195         Privacy: usersyncPrivacy{
196             gdprPermissions:  gdprPerms,
197             ccpaParsedPolicy: ccpaParsedPolicy,
    +           gppParsedPolicy:  gppParsedPolicy,
198         },
199         SyncTypeFilter: syncTypeFilter,
200     }
endpoints/cookie_sync.go
```
Discussed offline

**Review by guscarreon — DISMISSED** [review]
LGTM

**Comment by SyntaxNode** [comment]
Looks great. Thank you for working on this change. There are some areas which I recommend be updated in this PR:

`config/config.go`
- Please update the default value of `user_sync.redirect_url` to include the GPP and GPPSID macros before the `f=` parameter to keep a nice grouping.. and its important for the last macro to be the uid. This is for the callback to PBS.

`config/bidderinfo.go`
- The comment for the `SyncerEndpoint` type includes an example url. It's already long. What's your thoughts on including the new GPP and GPPSID macros?
- There is a detailed comment on the `SyncerEndpoint.URL` field which documents the available macros. Could you please add GPP and GPPSID macros there?
- There is a comment on `SyncerEndpoint.RedirectURL` which mentions the default value. Please update to include the new default including GPP and GPPSID.

`usersync/syncer_test.go`
- Please update the `TestBuildTemplate` test case `All Composed Macros` to include GPP and GPPSID. Update any other test you think should include GPP and GPPSID.

File: privacy/gpp/gpp.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package gpp
+
+// Policy represents the GPP privacy string container.
+// Currently just a placeholder until more expansive support is made.
+type Policy struct {
+	String string
+	RawSID string
```
[the remark this answers is not in the document]
Comment (hhhjort) [reply privacy/gpp/gpp.go:7]: I have decided on an inline comment. Changing `RawSID` would be something like `SIDCSV` or `CSVSID`, which just looks messy.

**Review by SyntaxNode — APPROVED** [review]

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**